src/main/java/com/minecolonies/coremod/entity/ai/citizen/cook/EntityAIWorkCook.java

```diff
@@ -324,6 +421,11 @@ private AIState cookFood()
         final TileEntity entity = world.getTileEntity(walkTo);
         if (entity instanceof TileEntityFurnace)
         {
+            // TODO: if the furnace isn't cooking food, we should complain about that
+
+            // TODO: if the furnace is no longer able to accept our uncooked food, should we keep feeding it fuel?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)